examples/demo_app/Gemfile

```diff
@@ -6,33 +6,25 @@ gem 'rails', '~> 6.0.0'
 # Integrate rails asset helpers with an external asset pipeline
 gem 'external_asset_pipeline', path: '../..'
 # Use sqlite3 as the database for Active Record
-gem 'sqlite3'
+gem 'sqlite3', '~> 1.4'
 # Use Puma as the app server
 gem 'puma', '~> 3.11'
 # Use SCSS for stylesheets
-gem 'sass-rails', '~> 5.0'
-# Use Uglifier as compressor for JavaScript assets
-gem 'uglifier', '>= 1.3.0'
```

Comment:
`uglifier` is removed because Rails 6 by default uses a nodejs pipeline (via `webpacker`) to compile all javascript. Our app doesn't need `uglifier` either, since it also uses a nodejs pipeline (via `external_asset_pipeline`) to compile javascript, so this is fine.